### DISCUSSION TITLE
Can ApplyOutOfSyncOnly=true be applied on ApplicationSet syncOptions?

### DISCUSSION THREAD

**Discussion by gajus** [argoproj/argo-cd#15937] [Q&A]
From reading the documentation, it is not clear whether I can apply `ApplyOutOfSyncOnly=true` on the `ApplicationSet` or if I need to create an Application that controls my ApplicationSet and apply `ApplyOutOfSyncOnly=true` to it?

And if I apply it to the top Application, does the change propagate to all other apps?

**Comment by ITBeyder** [comment]
it does for me..
